zkevm-circuits/src/lib.rs

```diff
@@ -16,6 +16,8 @@
 #![deny(missing_docs)]
 #![deny(unsafe_code)]
 #![deny(clippy::debug_assert_with_mut_call)]
+#![allow(deprecated)]
+#![allow(unreachable_code)]
```

Comment:
Suppressed deprecated code so they don't create noises in the cargo build.

unreachable_code was caused by those `todo!`s. We allow unreachable_code for now